apps/aecore/src/aec_accounts.erl

```diff
@@ -222,4 +259,16 @@ serialize_for_client(#account{id      = Id,
         end,
     maps:merge(ExtraInfo, #{<<"id">>      => aeser_api_encoder:encode(id_hash, Id),
                             <<"balance">> => Balance,
+                            <<"payable">> => is_payable(Account),
```

Comment:
Luca would ask: is there a corresponding release note :-).

Reply:
Luca would be correct, as usual. Will do that in a separate PR.